MichaelAquilina/flake8-spellcheck
# Updates to dictionary lists

- Create dedicated pytest list
- Add some new entries to the python list:
  - 'checkable' comes from '@runtime_checkable', a decorator introduced.
    in python3.8 and also available in typing_extensions.
  - 'exc' comes from the default argument names for '__exit__'
    functions.
  - 'tokenize' is a stdlib module used by this package.
  - 'unittest' is a stdlib module.
- Remove a couple of duplicate entries from the technical list.
- Add some new entries to the technical list:
  - 'filesystem' should be obvious.
  - 'func' is common shorthand for 'function'.
  - 'lua' is the name of an embeddable programming language.
  - 'runtime' is a common word to differentiate from e.g. compile-time.
    It's also used in the '@runtime_checkable' decorator.
  - 'ttl' stands for Time-to-Live, and is a common acronym seen in many
    places, including DNS and caching.
  - 'util' is a commonly used name in programming for a file full of
    utility functions.
- Removed some entries from this package's custom allowlist that are
  already specified elsewhere.

More from #55.
